rasa/nlu/selectors/response_selector.py

```diff
@@ -298,12 +303,16 @@ def preprocess_train_data(self, training_data: TrainingData) -> RasaModelData:
                 "all retrieval intents."
             )
 
+        label_attribute = RESPONSE if self.train_on_text else INTENT_RESPONSE_KEY
```

Comment:
Minor: Can we somehow unify the names `RESPONSE` and `INTENT_RESPONSE_KEY`? If you think it does not make sense it is also fine.

Reply:
But `label_attribute` is actually intended to unify them for the later part of the code, so they are just used here in this one line.